coordinator/src/main/scala/filodb.coordinator/DatasetCoordinatorActor.scala

```diff
@@ -168,6 +168,7 @@ private[filodb] class DatasetCoordinatorActor[K](projection: RichProjection[K],
       logger.warn("This should not happen. Not starting flush; $curReprojection; $flushingTable")
       return
     }
```

Comment:
Sorry, am confused what you're referencing.... those lines are from `startFlush` which has a signature return of `Unit`.